samples/csharp_dotnetcore/23.facebook-events/FacebookModel/FacebookMessage.cs

```diff
@@ -3,7 +3,7 @@
 
 using Newtonsoft.Json;
 
-namespace Facebook_Events_Bot.FacebookModel
+namespace Microsoft.BotBuilderSamples.FacebookModel
 {
```

Comment:
Missing cref in IsEcho property below, consider adding a proper link or removing the cref from /// comments.

Reply:
added link to facebook docs, thanks!